README.md

```diff
@@ -150,10 +159,7 @@ N/A
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-Original design idea by Cindy Shin in Behance.<a href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015.">BEHANCE</a>
-
-Icons by Surang (<a href="https://www.flaticon.com/br/icones-gratis/agricultura-e-jardinagem" title="agricultura e jardinagem ícones">Agricultura e jardinagem ícones criados por surang - Flaticon</a>)
-
+We are gratefull of Microvers stuff and founder for creating such an amazing envirnment that we can work and learn.
 
 ## 📝 License <a name="license"></a>
```

Comment:
- [ ] [OPTIONAL] There is a License link in your repo but the link is broken. Please make sure to fix broken link to make your readme more descriptive and more professional ✔️